modules/ossm-threescale-webassembly-module-examples-for-credentials-use-cases.adoc

```diff
@@ -0,0 +1,236 @@
+// Module included in the following assembly:
+//
+// service_mesh/v2x/ossm-threescale-webassembly-module.adoc
+
+[id="ossm-threescale-webassembly-module-examples-for-credentials-use-cases_{context}"]
+= The 3scale WebAssembly module examples for `credentials` use cases
+
+You will spend most of your time applying configuration steps to obtain credentials in the requests to your services.
+
+The following are `credentials` example, which you can modify to adapt to specific use cases.
+
+You can combine them all, although when you specify multiple source objects with their own `lookup queries`, they are evaluated in order until one of them successfully resolves.
+
+[id="api-key-in-query-string-parameters_{context}"]
+== API key (`user_key`) in query string parameters
+This looks up a `user_key` in a query string parameter or header of the same name:
```

Comment:
Here it starts with "This looks up" and for the next example it starts with "The following example looks up". Suggest syncing up on one or the other for consistency.